4 дз/4 дз/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿//Math
+
+using System;
+
+namespace Mathematics
+{
+
+    class Program
+    {
+        public static void Main()
+        {
+            Console.WriteLine("Введите х");
+            double x = Convert.ToDouble(Console.ReadLine());
+            double function = (Math.Sqrt(x + (1 / (Math.Sqrt(Math.Pow(x, 2) + 4)))));
```

Comment:
Квадрат числа проще вычислять через умножение
```suggestion
            double function = (Math.Sqrt(x + (1 / (Math.Sqrt(x * x + 4)))));
```